database/schema.sql

```diff
@@ -113,4 +113,245 @@ CREATE TABLE image (
     PRIMARY KEY (image_id),
     FOREIGN KEY (region_id) REFERENCES region(region_id),
     FOREIGN KEY (constellation_id) REFERENCES constellation(constellation_id)
-);
\ No newline at end of file
+);
+
+INSERT INTO aurora_colour (aurora_colour_id, colour, description, meaning) VALUES
```

Comment:
Great seed data and descriptions. We all looked at the AuroraWatch API and know that the UK can only be considered in one state at a given time, this expresses that well.